### PR TITLE
[TB-3061] Add reader mode font settings missing elements

### LINKED ISSUE #30: Change TextStyle nullability to non-null
It is contra intuitive to expose the Textstyle in the XStyles class as nullable `?`  . All textsyles are depended on the use of `Theme.headline1` which it self can be nullable, but as a user of Linden I expect that Theme.headline is set, otherwise I would expect a message from the system telling me to set it (like a crash - assertNotNull)

### PR DESCRIPTION
**Pull request by MichaelKMalak**
### What 🕵️ 🔍

- make text styles not nullable in linden by having a fallback TextStyle (fixes https://github.com/xaynetwork/xayn_design/issues/30)
- add missing reader mode font (noto serif font)
- add reader mode colors and text styles

### References 📝 🔗

- [TB-3061](https://xainag.atlassian.net/browse/TB-3061)

### CODE REVIEW

File: lib/src/linden/xstyles.dart [hunk 1]
```diff
@@ -21,156 +25,207 @@ class XStyles {
   FontWeight get _weightRegular => FontWeight.w400;
 
   /// Defined by design
-  TextStyle? get appScreenHeadline => _baseTextTheme.headline1?.copyWith(
+  TextStyle get appScreenHeadline =>
+      _baseTextTheme.headline1?.copyWith(
+        color: colors.primaryText,
+        fontWeight: _weightBold,
+        fontSize: 24.0,
+        // height = 32
```
Comment (carminetoriello) [inline lib/src/linden/xstyles.dart:33]: ❓ Do we still need this commented line ?

Comment (MichaelKMalak) [reply lib/src/linden/xstyles.dart:33]: It’s more of a helper to point that 1.33 height is actually 32 pixels, since in figma height is in pixels.

Comment (carminetoriello) [reply lib/src/linden/xstyles.dart:33]: I see, thanks!

**Review by ghost — APPROVED** [review]
🚀

**Pull request merged**